app/views/declaration/summary/countries_section.scala.html

```diff
@@ -30,6 +30,11 @@
     locations.originationCountry.isDefined || locations.hasRoutingCountries.isDefined || locations.routingCountries.nonEmpty || locations.destinationCountry.isDefined
 }
 
+@actions(action: Actions) = @{
```

Comment:
:+1:

Reply:
This was a bit of a fudge so that I could replace Some(Action) with actions(Action) - i.e. to minimise the changes to each section of the summary page.